src/components/ToolbarDropdown/ToolbarDropdown.tsx

```diff
@@ -8,7 +8,7 @@ type ToolbarDropdownProps = {
   nameDropdown?: string;
   initialValue: number | string;
   initialLabel: string | undefined;
-  onClick: PropTypes.func;
+  handleSelect: PropTypes.func;
```

Comment:
Just FYI: PropTypes are not needed with Typescript unless you are creating libraries consumed by js libraries. Since all of our stuff is Typescript it is not necessary. There is also a runtime penalty for using PropTypes (both runtime size and execution time).
https://stackoverflow.com/questions/41746028/proptypes-in-a-typescript-react-application

The exception is `PropTypes.context`

Reply:
This has been a confusing issue for me as well.

Reply:
The problem was that the drop-down inside toolbardropdown component  has {...this.props} so drop-down had onChange and OnClick property.

Reply:
About proptype @mtho11 I think that we should use them (I wrote about in the guide). I don't know what do you think about use proptype, flux or none but we shouldn't use proptype a in some components and no in others, it should be consistent in all the app (my opinion)

Reply:
From my understanding, PropTypes are not necessary with Typescript. I'll send out an email and we can discuss more. Maybe there is something I'm missing.